_layouts/default.html

```diff
@@ -73,6 +71,9 @@ <h1>{{ site.title | default: site.github.repository_name }}</h1>
 
     <footer>
       <div class=social>
+        <a href=mailto:info@emissions-api.org
+        title=Email
+        class="fas fa-at"></a>
```

Comment:
- indentation
- not sure but maybe [fa-envelope](https://fontawesome.com/icons/envelope?style=solid) would look better.
It feels to me like the @-sign was used in the 1990th and got replaced by the ✉-sign after 2000 ;-)

Reply:
I totally agree with you. The reason why I used the @ in the end, was that I thought that this is clearer, since I could also use the envelope for postal address. Maybe I am just old and everybody else is totally clear that an envelope is an email.

Reply:
But the indentation should be fixed by now.

Reply:
> Let's just discuss this a little bit more and then probably use the envelope in the end smile

Okay, then let's just jump ahead, assume the discussion to be done and use `fa-envelope`.

Reply:
I used the envelope.